refs.h

```diff
@@ -523,7 +523,7 @@ int for_each_reflog(each_ref_fn fn, void *cb_data);
 #define REFNAME_REFSPEC_PATTERN 2
 
 /*
- * Return 0 iff refname has the correct format for a refname according
+ * Return 0 if refname has the correct format for a refname according
```

Comment:
This was not a typo. "iff" is a synonym for "if and only if". See e.g. [this definition](https://www.merriam-webster.com/dictionary/iff).

Reply:
Ok, Fixed.